diff --git a/Makefile b/Makefile
index 36a451c..4a63660 100644
--- a/Makefile
+++ b/Makefile
@@ -3,8 +3,7 @@ CUDNN=0
 OPENCV=0
 DEBUG=0
 
-ARCH= -gencode arch=compute_20,code=[sm_20,sm_21] \
-      -gencode arch=compute_30,code=sm_30 \
+ARCH= -gencode arch=compute_30,code=sm_30 \
       -gencode arch=compute_35,code=sm_35 \
       -gencode arch=compute_50,code=[sm_50,compute_50] \
       -gencode arch=compute_52,code=[sm_52,compute_52]
